apps/extension/src/approve-origin.ts

```diff
@@ -4,8 +4,8 @@ import { errorFromJson } from '@connectrpc/connect/protocol-connect';
 import { localExtStorage } from '@penumbra-zone/storage';
 import { OriginApproval, PopupType } from './message/popup';
 import { popup } from './popup';
-import Map from '@penumbra-zone/polyfills/Map.groupBy';
-import { UserChoice } from '@penumbra-zone/types/src/user-choice';
+import Map from '@penumbra-zone/polyfills/src/Map.groupBy';
```

Comment:
All polyfill files moved into /src folder for organizational purposes (did this with a few packages). Most of the time created barrel files, but it doesn't really fit so well with polyfills given some of them produce side effects upon importing.